app/src/androidTest/java/com/github/orkest/View/profile/ProfileUITest.kt

```diff
@@ -43,7 +43,9 @@ class ProfileUITest {
         )
 
         composeTestRule.setContent {
-            OrkestTheme { topProfile(viewModel = viewModel) }
+            ProfileActivitySetting {
+                ProfileActivityScreen(ProfileActivity(), viewModel = viewModel)
+            }
         }
     }

```

Comment:
This line of code will have to be added in each of your test functions to avoid merge conflicts